src/components/leads/Leads.html

```diff
@@ -0,0 +1,19 @@
+<table class="table table-bordered" aria-describedby="table of leads">
+  <caption>
+    Table of Leads
```

Comment:
Please move the hardcoded title of the table and the headers like you did previously with the hardcoded **CRM** text.

Reply:
Changes done.. thank you

Reply:
We will update this later, it is OK to extract it to the class but it does not make sense to read it from an env. var.